src/main/java/spireMapOverhaul/zones/gravewoodGrove/powers/DeadBranchPower.java

```diff
@@ -0,0 +1,53 @@
+package spireMapOverhaul.zones.gravewoodGrove.powers;
+
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.common.RelicAboveCreatureAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.RelicLibrary;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import spireMapOverhaul.SpireAnniversary6Mod;
+
+public class DeadBranchPower extends AbstractGravewoodGrovePower {
+
+    public static final String POWER_ID = SpireAnniversary6Mod.makeID("DeadBranch");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    private int cardsExhaustedThisTurn = 0;
+
+
+    public DeadBranchPower(AbstractCreature owner) {
+        super(POWER_ID, NAME, AbstractPower.PowerType.DEBUFF, false, owner, 0);
+    }
+
+    public void onExhaust(AbstractCard card) {
+        if (!AbstractDungeon.getMonsters().areMonstersBasicallyDead()) {
+            flash();
+            addToBot(new RelicAboveCreatureAction(AbstractDungeon.player, RelicLibrary.getRelic("Dead Branch")));
+            addToBot(new MakeTempCardInHandAction(AbstractDungeon.returnTrulyRandomCardInCombat().makeCopy(), false));
```

Comment:
The action itself already makes a copy of the card, but it's fine

Reply:
keeping it in as discussed